boards/arm/lpcxpresso55s36/doc/index.rst

```diff
@@ -66,6 +66,7 @@ The Zephyr configuration for the LPC55S36 board (lpcxpresso55s36) is as follows:
 | CLOCK     | on-chip    | clock_control                       |
 +-----------+------------+-------------------------------------+
 | CAN       | on-chip    | canbus                              |
+| PWM       | on-chip    | pwm                                 |
```

Comment:
Will this render correctly in the documentation? I think you need the ----- lines to properly render the table